assets/queries/k8s/ensure_administrative_boundaries_between_resources/query.rego

```diff
@@ -0,0 +1,26 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.k8s as k8sLib
+
+listKinds := ["Pod", "Deployment", "DaemonSet", "StatefulSet", "ReplicaSet", "ReplicationController", "Job", "CronJob", "Service", "Secret", "ServiceAccount", "Role", "RoleBinding", "ConfigMap", "Ingress"]
+
+CxPolicy[result] {
+	resource := input.document[i]
+	metadata := resource.metadata
+
+    resource.kind == listKinds[_]
+    namespace := metadata.namespace
+
+    c := count({j | res = input.document[j]; res.kind == listKinds[_]; res.metadata.namespace == namespace})
+    c > 1
```

Comment:
Not sure if this approach is the best. 

When a namespace is not defined, by default, the namespace is "default", which does not fit the best practice of limiting the scope. Maybe we should advise the users to not use a default namespace.

Additionally, I think that using the same namespace more than once is not a vulnerability.  As mentioned in the CIS WorkBench Kubernetes documentation, "Ensure that these namespaces are the ones you need and are adequately administered as per your requirements".  I think we should advise the user to review its namespaces, in case the user has many and in case the user has a few.


Anyway, please be careful with the typos in `keyExpectedValue` and `keyActualValue` (probably it will be changed)